x-pack/plugins/security/server/authentication/can_redirect_request.ts

```diff
@@ -22,12 +23,13 @@ export function canRedirectRequest(request: KibanaRequest) {
   const route = request.route;
   const hasVersionHeader = headers.hasOwnProperty(KIBANA_VERSION_HEADER);
   const hasXsrfHeader = headers.hasOwnProperty(KIBANA_XSRF_HEADER);
+  const hasIternalOriginHeader = headers.hasOwnProperty(X_ELASTIC_INTERNAL_ORIGIN_REQUEST);
```

Comment:
```suggestion
  const hasInternalOriginHeader = headers.hasOwnProperty(X_ELASTIC_INTERNAL_ORIGIN_REQUEST);
```

Reply:
No longer implemented. Thanks for catching that!